apps/bfd-db-migrator/src/main/java/gov/cms/bfd/migrator/app/MigratorApp.java

```diff
@@ -19,8 +41,92 @@
    */
   public static void main(String[] args) {
     LOGGER.info("Successfully started");
-    LOGGER.info("Hello, World!");
+
+    AppConfiguration appConfig = null;
+    try {
+      appConfig = AppConfiguration.readConfigFromEnvironmentVariables();
+      LOGGER.info("Application configured: '{}'", appConfig);
+    } catch (AppConfigurationException e) {
+      System.err.println(e.getMessage());
+      LOGGER.warn("Invalid app configuration.", e);
+      System.exit(EXIT_CODE_BAD_CONFIG);
+    }
+
+    MetricRegistry appMetrics = setupMetrics(appConfig);
+
+    // Create a data source for use by createOrUpdateSchema
+    HikariDataSource pooledDataSource =
+        createPooledDataSource(appConfig.getDatabaseOptions(), appMetrics);
+
+    // run migration
+    boolean migrationSuccess =
+        DatabaseSchemaManager.createOrUpdateSchema(pooledDataSource, appConfig);
+
     LOGGER.info("Shutting down");
-    System.exit(0);
+    if (!migrationSuccess) {
+      System.exit(EXIT_CODE_FAILED_MIGRATION);
+    }
+    System.exit(EXIT_CODE_SUCCESS);
```

Comment:
❤️  This is super simple! I know we've recently disabled hibernate validations for a variety of reasons, but do you think that's going to be permanent? If not, is there room to add hibernate validations under this line of effort/PR, or should we raise a new issue to implement it in the near future?

Reply:
Trotter and I talked about it, and we'll likely do the hibernate validations in a different effort/story since it was not part of this original scope.